linux/install.sh

```diff
@@ -7,8 +7,14 @@ fc-list | grep "Bitstream Vera" > /dev/null
 RETURN=$?
 if [ $RETURN -ne 0 ];then
   echo "Bitstream Vera font family not found. Please install it:"
-  echo "sudo apt-get install ttf-bitstream-vera"
-  exit 1
+	# Detects if using Ubuntu or openSUSE, then install.
+	if [[ "$(. /etc/os-release; echo $NAME)" =~ "Ubuntu" ]]; then
```

Comment:
This would work on my work machine, but my personal machine runs Mint.
```
$ cat /etc/os-release 
NAME="Linux Mint"
```
I also want to support for all other Debian/Ubuntu derivatives. Check for openSUSE, but then fallback to Ubuntu.